frontend/Dockerfile

```diff
@@ -2,7 +2,7 @@
 FROM registry.access.redhat.com/ubi9/nodejs-20:latest as builder
```

Comment:
Should we name this `Containerfile`?

Reply:
Yes https://github.com/containers/buildah/discussions/3170#discussion-3330099

Reply:
On the previous PR I missed a tag 
 io.openshift.tags="operator,trustification,trustify,ui,nodejs18" \
 
 should be:
 
 io.openshift.tags="operator,trustification,trustify,ui,nodejs20" \

Reply:
> Should we name this `Containerfile`?

We can, and I've renamed it

> Should we name this `Containerfile`?

Done